### PR TITLE
ci: Use upstream vale action

### PR DESCRIPTION
**Pull request by jimmidyson**
### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-<add_pr_##_here>.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Comment by juliangieseke** [comment]
seems like vale isnt running (properly). its green despite the changes being red in their own PR #4334

File: .github/workflows/vale.yml [hunk 1]
```diff
@@ -8,8 +8,8 @@ jobs:
     continue-on-error: true
     steps:
     - name: Checkout
-      uses: actions/checkout@master
-    - uses: pierrebeitz/vale-action@ce94dd2b4ef576bf1237f45bf2bbd9ed4877e740
+      uses: actions/checkout@v3
+    - uses: errata-ai/vale-action@v1
       with:
         onlyAnnotateModifiedLines: true
```
Comment (juliangieseke) [inline .github/workflows/vale.yml:14]:
```suggestion
```

looks like pierre added that in his fork? I doubt its crashing the whole thing but who knows 🤷

**Review by vacas5 — COMMENTED** [review]
can we commit this separately from content changes so we can revert easily if need be?

**Comment by juliangieseke** [comment]
@vacas5 it was rebased onto a content change to test if it works - it doesnt. :( (check lint action, it didnt do anything)

**Comment by juliangieseke** [comment]
I think we can close this for now.

**Pull request closed without merging**